addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -88,3 +89,39 @@ def test_notify_reply_to_computation_mc(self):
                     "%s %s" % (self.user_employee_c2.company_id.name, test_record.name),
                     "%s@%s" % (self.alias_catchall, self.alias_domain)))
             )
+
+@tagged('multi_company')
+class TestMultiCompanySetupAttachment(TestMailCommon, TestRecipients):
```

Comment:
Can't we merge it into "TestMultiCompanySetup" class as currently we have few tests ?